internal/cloud/atlas/clusters.go

```diff
@@ -0,0 +1,44 @@
+package atlas
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+// Cluster contains non sensitive data about an Atlas cluster
+type Cluster struct {
+	ID    string `json:"id"`
+	Name  string `json:"name"`
+	State string `json:"stateName"`
+}
+
+const (
+	clustersByGroupIDPattern = "/api/atlas/v1.0/groups/%s/clusters"
```

Comment:
I noticed for the other endpoints the route is something like "/api/public/v1.0", which wasn't working getting a 404. So switched to what the docs point to.

Reply:
yeah i noticed this `public` vs `atlas` discrepancy when first implementing the atlas client, and stuck with what we previously used.

i think this is fine here, but maybe add `atlasAPI = "/api/atlas/v1.0"` in the `client` file?